test/index.js

```diff
@@ -63,6 +71,32 @@ describe('Klaviyo', function () {
         .end(done);
     });
 
+    it('should send the Placed Order event correctly for Completed Order event', function(done){
```

Comment:
"correctly" doesn't have a clear definition here. If you nest this inside a `completedOrder` describe, you can rephrase to:

`should successfully send a Placed Order event`